.gitlab-ci.yml

```diff
@@ -145,24 +199,3 @@ build_documentation:
   variables:
     WHEEL_PATH: build/dist/coremltools*cp39-none-macosx_10_15_x86_64.whl
     PYTHON: "3.9"
-
-#########################################################################
-##
-##                       Collect artifacts
-##
-#########################################################################
-collect_artifacts:
-  tags:
-    - macos12
-  stage: collect
-  script:
-    echo "Collect artifacts (wheels and documentation)"
-  dependencies:
-    - build_wheel_macos_py37
-    - build_wheel_macos_py38
-    - build_wheel_macos_py39
-    - build_documentation
-  artifacts:
-    expire_in: 2 weeks
-    paths:
-      - build/dist/
```

Comment:
Why are artifacts no longer necessary?

Reply:
We've actually never used the artifacts here.